### ISSUE TITLE
install problem

### ISSUE THREAD

**Issue by elad-eyal** [wsdjeg/vim-fetch#31]
is it intentional that sourceing v3.0.0 vimball creates the files under `~/.vim/Users/martin` ? (I'm Using  Windows , WSL, vim 8.0)

**Comment by kopischke** [comment]
Nope; that’s on me messing up Vimball generation while I was still the maintainer. As I’m not anymore, all I can do is apologise and defer to @wsdjeg.

**Comment by elad-eyal** [comment]
You can create the vimball inside a "github workflow" nowadays. It's quite nice.

I can confirm that removing the string `/Users/martin/Werkbank/Projekte kopischke/vim-fetch/` from line start the tarball made things install well for me (using `source %`)

**Comment by wsdjeg** [comment]
I do not see any vimball script in this repo, what can be done in this repo?

**Comment by kopischke** [comment]
@wsdjeg there are Vimballs for download on the release page.

**Comment by elad-eyal** [comment]
For example I thing you can see this page https://medium.com/faun/automate-your-build-release-with-github-actions-367c0febf5fd


Automatic creation of the vimball for vthe release

**Comment by wsdjeg** [comment]
Ok, Thanks, I will check it.

**Comment by elad-eyal** [comment]
@wsdjeg ?

**Comment by EvanHahn** [comment]
Just ran into this issue. Before installing, removing the prefix (by running `:%s/\/Users\/martin\/Werkbank\/Projekte kopischke\/vim-fetch\//`) fixed my problem.

**Comment by wsdjeg** [comment]
I have deleted the vimball file, and I will use the googleapi/please-release action to generate new release.
